edx_rest_framework_extensions/auth/jwt/tests/utils.py

```diff
@@ -23,7 +23,7 @@ def generate_jwt_token(payload, signing_key=None):
     Generate a valid JWT token for authenticated requests.
     """
     signing_key = signing_key or settings.JWT_AUTH['JWT_ISSUERS'][0]['SECRET_KEY']
-    return jwt.encode(payload, signing_key).decode('utf-8')
+    return jwt.encode(payload, signing_key)
```

Comment:
https://github.com/jpadilla/pyjwt/pull/513/files#diff-08a732a5481e696aabf94020c0ad5b788ceab96dc755c9f1bfcf7efec9204ff2R131

Reply:
This is a backward incompatible change that should get a major version bump and be described in the README. Reminder to use a `!` in the conventional commit comment.

I imagine you will upgrade edx-platform (what you found) immediately? https://github.com/search?q=org%3Aedx+generate_jwt&type=code

Reply:
Reminder for commit comment as well. ^